VERSION.cmake

```diff
@@ -1,3 +1,4 @@
-SET(HAWKEY_MAJOR "0")
-SET(HAWKEY_MINOR "5")
-SET(HAWKEY_PATCH "4")
+SET(HAWKEY_VERSION "0.5.3")
```

Comment:
The ```3``` is intentional? This is going to break CI since the current DNF requires hawkey >= 0.5.4